cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -734,6 +734,18 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.sql.jdbc.driver.external</name>
+    <value>true</value>
+    <description>
+      Indicates whether the JDBC driver has to be loaded from an external directory.
+      If true, then the JDBC driver directory has to be specified using
+      "data.storage.sql.jdbc.driver.directory".
```

Comment:
`data.storage.sql.jdbc.driver.name` also needs to be specified if this is set to true.

Reply:
Added it to the definition of `data.storage.sql.jdbc.driver.directory`